src/ttsim/params-schema.json

```
@@ -0,0 +1,91 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "patternProperties": {
+      "^[a-zA-Z0-9äöüßÄÖÜ_]+$": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "object",
+            "properties": {
+              "de": { "type": "string" },
+              "en": { "type": ["string", "null"] }
+            },
+            "required": ["de"],
+            "additionalProperties": false
+          },
+          "description": {
+            "type": "object",
+            "properties": {
+              "de": { "type": "string" },
+              "en": { "type": ["string", "null"] }
+            },
+            "required": ["de"],
+            "additionalProperties": false
+          },
+          "unit": {
+            "type": ["string", "null"],
+            "enum": [
+              "Euro",
+              "DM",
+              "Share",
+              "Percent",
+              "Factor",
+              "Year",
+              "Month",
+              "Hour",
+              "Square Meter",
+              "Euro / Square Meter",
+              null
+            ]
+          },
+          "type": {
+            "type": "string"
+          },
+          "reference_period": {
+            "type": "string",
+            "enum": ["Year", "Quarter", "Month", "Week", "Day"]
+          },
+          "access_prior_parameters": {
+            "type": "object",
+            "properties": {
+              "reference_period": {
+                "type": "string",
+                "enum": ["Year", "Quarter", "Month", "Week", "Day"]
+              },
+              "number_of_lags": {
+                "type": "integer",
+                "minimum": 1
+              }
+            },
+            "required": ["reference_period", "number_of_lags"],
+            "additionalProperties": false
+          }
+        },
+        "patternProperties": {
+          "^(19|20|21)[0-9]{2}-[0-9]{2}-[0-9]{2}$": {
```

```suggestion
          "^(19|20|21)[0-9]{2}-(0[1-9]|1[0-2])-[0-9]{2}$": {
```

Month should not be larger than 12